src/org/usfirst/frc4904/autonly/RightSideTime.java

```diff
@@ -1,9 +1,18 @@
 package org.usfirst.frc4904.autonly;
 
 import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.standard.commands.Noop;
 
 public class RightSideTime extends SideSwitchTime {
 	public RightSideTime () {
-		super(RobotMap.gamefield.ourSwitch.right);
+		super();
+	}
+	
+	@Override
+	public void initialize() { //TODO: Replace with actual deployment code.
```

Comment:
This hasn't been updated to fit pattern of the `LeftSideTime` which has been tested. Please fix. WPILib gets mad about addSequential or anything of the sort post-initialize.

Reply:
This also won't work because you're overriding the initialize of the `Command` superclass as opposed to the `setup` method of the underlying `Strategy`